pdr_backend/sim/test/test_sim_engine.py

```diff
@@ -104,3 +109,137 @@ def test_sim_engine(tmpdir, check_chromedriver, dash_duo):
         assert "tab--selected" in tab.get_attribute("class")
         for figure_name in figures:
             dash_duo.find_element(f"#{figure_name}")
+
+
+def _clear_test_db(ppss):
```

Comment:
don't we already have this as a fixture?

Reply:
I don't think we do